risc0/zkvm/src/host/client/prove/bonsai.rs

```diff
@@ -81,13 +81,18 @@ impl Prover for BonsaiProver {
         let session = client.create_session(image_id_hex, input_id, receipts_ids, false)?;
         tracing::debug!("Bonsai proving SessionID: {}", session.uuid);
 
+        // TODO(#1759): Improve upon this polling solution.
+        let polling_interval = if let Ok(ms) = std::env::var("BONSAI_POLL_INTERVAL_MS") {
```

Comment:
I find seconds to be a more natural unit than milliseconds to put in places like env vars and JSON. And then the default value is simply `1` (previously `5`).

Reply:
The issue is we want to support sub-second polling times for some low-latency use cases. How were you thinking about handling that? This be a floating point value?

Reply:
Yeah I think 1e-3 represented with 53 bits of floating precision is fine

Reply:
I'm unopinionated, this shouldn't be a commonly used env var anyway, if 👍 given by those who have already reviewed are given, happy to make that change

Reply:
Just something to consider, not an r-